Orange/widgets/data/owsql.py

```diff
@@ -124,17 +126,23 @@ def __init__(self):
         box.layout().addWidget(self.passwordtext)
 
         self._load_credentials()
+        self.tablemodels = TableModels()
 
         tables = gui.hBox(box)
-        self.tablemodel = TableModel()
         self.tablecombo = QComboBox(
             minimumContentsLength=35,
             sizeAdjustPolicy=QComboBox.AdjustToMinimumContentsLength
         )
-        self.tablecombo.setModel(self.tablemodel)
+        self.tablecombo.setModel(self.tablemodels)
         self.tablecombo.setToolTip('table')
         tables.layout().addWidget(self.tablecombo)
-        self.tablecombo.activated[int].connect(self.select_table)
+        self.connect()
+
+        index = self.tablecombo.findText(str(self.table))
+        if index != -1:
+            self.tablecombo.setCurrentIndex(index)
+        self.tablecombo.activated[int].connect(self.select_table) # set up the callback to select_table in case of selection change
```

Comment:
Could you move the commend in a line before the command? We try to limit the lines in source code to maximum of 100 characters (though some go even further and limit it to 79)